packages/amplify-util-uibuilder/src/__tests__/handleEvents.test.ts

```diff
@@ -0,0 +1,18 @@
+describe('can handle events', () => {
+  let context: any;
+  beforeEach(() => {
+    context = {
+      amplify: {
+        invokePluginMethod: () => true,
+      },
+    };
+  });
+  it('handles postEnvAdd', async () => {
+    const { run } = require('../event-handlers/handle-PostEnvAdd');
+    await run(context);
```

Comment:
should there be some assertions after running? (also applies to other tests that run but don't assert anything)